### DISCUSSION TITLE
Why not use a pin from https://trakt.tv/pin/478 ?

### DISCUSSION THREAD

**Discussion by danielb2** [Taxel/PlexTraktSync#1729] [Support] [answered]
Is there any reason there's not a setup just using the auth from https://trakt.tv/pin/478 ?

**Comment by simonc56** [comment, marked as answer]
Yes, there is one : deprecated since 2016.

**Reply by danielb2** [reply]
oh but it still works with emby and plex's original trakt plugin. the trakt plugin works fine for linux still, just not macos

**Reply by danielb2** [reply]
@simonc56  ?

**Reply by simonc56** [reply]
Question has been answered. What do you want me to say ?

**Reply by danielb2** [reply]
No. You didn't actually answer the question. You didn't say why not to use that. It's clearly works. Is it a difficult question to answer?

**Reply by simonc56** [reply]
I can repeat the answer : because it's deprecated.
We do not implement deprecated API.

**Reply by glensc** [reply]
also, it's unclear who owns the app "trakt (for Plex)". might be not wise security wise authorize to app whose owner is unknown.

but counter question: why should we?

**Reply by glensc** [reply]
Would you feel secure authorizing to random app? Can I steal your identity?

- https://trakt.tv/pin/666

(I don't know to whom this belongs)
